Dockerfile

```diff
@@ -125,23 +125,20 @@ RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
 FROM build as pythonbuild
 
 ARG PYTHON_VERSION=3.10.6
-ARG PYTHON2_VERSION=2.7.18
 ARG PYENV_GIT_TAG=v2.3.4
 
 ENV PYENV_ROOT=/opt/python
 RUN curl -kSs https://pyenv.run | bash
 ENV PATH=${PYENV_ROOT}/shims:${PYENV_ROOT}/bin:$PATH
-RUN for pyvers in ${PYTHON_VERSION} ${PYTHON2_VERSION}; do \
-    pyenv install -v $pyvers; done
-RUN pyenv global ${PYTHON_VERSION} ${PYTHON2_VERSION}
+RUN pyenv install -v ${PYTHON_VERSION}
+RUN pyenv global ${PYTHON_VERSION}
```

Comment:
This hint from Lift actually sounds sensible, and we should probably do

    RUN pyenv install -v ${PYTHON_VERSION} && pyenv global ${PYTHON_VERSION}

Reply:
I guess that's right, but I'm about to remove those lines anyway as we don't need a custom Python version anymore. I just didn't have time to make that commit yet.

Reply:
Will that commit come to this PR?

Reply:
no

Reply:
Then I'd prefer to have the `RUN` statements consolidated to not leave it in this state intermediately.

Reply:
This PR did not introduce the issue, it's a separate commit anyway, so why should it have to be part of this PR?

Reply:
Well, in my view your change at least made it more obvious that these lines should be consolidated now; the syntax would have been a bit awkward before. So I thought it's a sensible thing to do, and it would have been much quicker than having this discussion. But I see your point that very strictly speaking that change is out of scope of your change.